app/controllers/cancelRegistration/CancelRegistrationCYAController.scala

```diff
@@ -26,18 +27,14 @@ import views.html.cancelRegistration.CancelRegistrationCYAView
 
 class CancelRegistrationCYAController @Inject()(
                                             override val messagesApi: MessagesApi,
-                                            identify: IdentifierAction,
-                                            getData: DataRetrievalAction,
-                                            requireData: DataRequiredAction,
+                                            controllerActions: ControllerActions,
                                             val controllerComponents: MessagesControllerComponents,
                                             view: CancelRegistrationCYAView
                                           ) extends FrontendBaseController with I18nSupport {
 
-  def onPageLoad(): Action[AnyContent] = (identify andThen getData andThen requireData) {
+  def onPageLoad(): Action[AnyContent] = controllerActions.withRequiredJourneyData(SelectChange.CancelRegistration) {
```

Comment:
It's a bit cleaner to import SelectChange in all the controllers requiring the type and remove the Class prepended before the change type.
`def onPageLoad(): Action[AnyContent] = controllerActions.withRequiredJourneyData(CancelRegistration)`